doc/oob_streams.md

```diff
@@ -42,14 +42,28 @@ public async Task TakeLargeFileAsync(IDuplexPipe pipe)
 }
 ```
 
+The server may also reply with stream or pipe:
+
+```cs
+public async Task<Stream> SendLargeFileAsync()
+{
+   var streamPair = FullDuplexPipe.CreatePair();
+   
+   // Use streamPair.Item1
+
+   // Dispose of the stream to notify the client we are done sending data
+   streamPair.Item1.Dispose();
+
+   return streamPair.Item2;
```

Comment:
I have to agree with you there. I instead replaced this with a FileStream that then gets forwarded to the client since that will definitely be a more common case than a two-way communication.